test/inductor/test_max_autotune.py

```diff
@@ -10,10 +10,7 @@
 from torch._inductor.select_algorithm import AlgorithmSelectorCache, ChoiceCaller
 from torch._inductor.virtualized import V
 from torch.fx.experimental.proxy_tensor import make_fx
-from torch.testing._internal.common_utils import (
-    instantiate_parametrized_tests,
-    parametrize,
-)
+from torch.testing._internal.common_utils import instantiate_parametrized_tests
```

Comment:
Since this is no longer a parameterized tests, we can remove this import and its usage as well.